stdlib/2/unittest.pyi

```diff
@@ -153,11 +152,22 @@ class TestSuite(Testable):
     def debug(self) -> None: ...
     def countTestCases(self) -> int: ...
 
-# TODO TestLoader
-# TODO defaultTestLoader
+class TestLoader:
+    def loadTestsFromTestCase(self,
+                              testCaseClass: Union[TestCase, None]) -> TestSuite: ...
+    def loadTestsFromModule(self, module: str = ...,
+                            use_load_tests: bool = ...) -> TestSuite: ...
+    def loadTestsFromName(self, name: str = ...,
+                          module: Optional[str] = ...) -> TestSuite: ...
+    def loadTestsFromNames(self, names: List[str] = ...,
+                          module: Optional[str] = ...) -> TestSuite: ...
```

Comment:
Can you add `getTestCaseNames` and the three attributes documented under https://docs.python.org/2/library/unittest.html#unittest.TestLoader.testMethodPrefix?